bobtemplates/plone/addon/setup.py.bob

```diff
@@ -46,7 +46,13 @@ setup(
     keywords='Python Plone',
     author='{{{ author.name }}}',
     author_email='{{{ author.email }}}',
-    url='https://pypi.python.org/pypi/{{{ package.dottedname }}}',
+    url='https://github.com/collective/{{{ package.dottedname }}}',
+    project_urls={
+        'PyPI': 'https://pypi.python.org/pypi/{{{ package.dottedname }}}',
+        'Source': 'https://github.com/collective/{{{ package.dottedname }}}',
```

Comment:
source and url the same? how does it display on pypi?

Reply:
one is homepage one is source, but the name source is more clear since homepage points to github in only half of the cases. I would keep it double.